### PR TITLE
remove windows support

### PR DESCRIPTION
**Pull request by janstary**
Current code does not build on windows.
In fact, the previous release (1.21) does not either.
So remove it for now, and for the upcomming 1.22

### CODE REVIEW

**Comment by janstary** [comment]
Note that there are still pieces of code trying to accommodate windows (`grep -F WIN32`).
This removes the obvious, so that we don't ship broken code; further cleanup needed.

File: README.md [hunk 1]
```diff
@@ -21,19 +21,17 @@ Refer to the individual manpages for details.
...
-to compile all the projects.
-
-* In Visual C++ 6.0, open workspace file rtptools.dsw.
-* In VC menu Build, use Batch Build to build all the tools.
-* All the rtptools will be created under the "debug\" directory.
+```
+aclocal
+autoconf
+automake
+```
```
Comment (mcd500) [inline README.md]: Could you make a new pull request only for the changes in README.md for the aclocal, autoconf, automake after the git pull, so I could apply this change first separately from Windows parts?

**Comment by janstary** [comment]
README edited in https://github.com/columbia-irt/rtptools/pull/24

**Comment by janstary** [comment]
This branch is already in conflict - I prepared a clean PR in https://github.com/columbia-irt/rtptools/pull/26 , closing this one

**Pull request closed without merging**